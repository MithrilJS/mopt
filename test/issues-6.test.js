"use strict";

var code = require("./lib/code");

describe("Issues", () => {
    describe("#6 quoted properties", () => {
        it("should be supported", () =>
            expect(
                code(`m("div", { "fooga" : 0 })`)
            )
            .toMatchSnapshot()
        );
    });
});
