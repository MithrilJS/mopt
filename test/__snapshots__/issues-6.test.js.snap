// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Issues #6 quoted properties should be supported 1`] = `"m.vnode(\\"div\\",undefined,{\\"fooga\\":0},[],undefined,undefined);"`;
